lib/active_model/serializer/configuration.rb

```diff
@@ -19,6 +19,7 @@ def config.array_serializer
           collection_serializer
         end
 
+        config.default_include_tree = '*'
```

Comment:
Can you think of a more descriptive name for this?

Future PR: would be a reasonable serializer config